metrics/lib/common.bash

```diff
@@ -92,6 +92,20 @@ check_images()
 	done
 }
 
+# This function performs a docker build on the image names
+# passed in, to ensure that we have the latest changes from
+# the dockerfiles
+check_dockerfiles_images()
+{
+	local image="$1"
+	local dockerfile_path="$2"
+	local exists=$(docker image inspect $image > /dev/null; echo $?)
+	if [ $exists -eq 0 ] || [ $exists -ne 0 ]; then
+		echo "Building Dockerfile Image : $image"
+		docker build --label "$image" --tag "${image}:latest" -< "$dockerfile_path"
```

Comment:
I've seen that docker builds does not repete the steps already done. So if you change the dockerfile will execute the dockerfile. Probably is better always build the image.